seqr/views/apis/report_api.py

```diff
@@ -342,7 +343,27 @@ def _process_saved_variants(saved_variants_by_family, family_individual_affected
             variant['inheritance_models'] = inheritance_models
             for gene_id in potential_compound_het_gene_ids:
                 potential_com_het_gene_variants[gene_id].append(variant)
+            for tag in variant['discovery_tag_names']:
+                for guid in variant.get('discovery_tag_guids_by_name', {}).get(tag):
+                    potential_mnvs[tag][guid].append(variant)
```

Comment:
the `guid` will always have the same tag name, so theres no need to do the nested dict - you can just do a regular `defaultdict(list)` and then here leave out the tag check and do `potential_mnvs[guid].append(variant)`